### PR TITLE
Docs/prisma commands on docs

### PR DESCRIPTION
**Pull request by miguelnietoa**
## What does this PR do?

Refs #1839, #1843, #1872

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code and corrected any misspellings
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by pumfleet — APPROVED** [review]

File: apps/docs/pages/self-hosting/upgrade.mdx [hunk 1]
```diff
@@ -3,41 +3,41 @@
...
 4. Start the server. In a development environment, just do:
-   ```
-   yarn dev
+   ```sh
+   yarn dev --scope=@calcom/web
    ```
    For a production build, run for example:
+   ```sh
+   yarn build --scope=@calcom/web
+   yarn start --scope=@calcom/web
```
Comment (zomars) [inline apps/docs/pages/self-hosting/upgrade.mdx:41]: Same here:

```suggestion
   yarn build
   yarn start
```

**Review by zomars — CHANGES_REQUESTED** [review]

**Comment by miguelnietoa** [comment]
Hi @zoomars, thanks for reviewing. The requested changes have been applied.

**Review by zomars — APPROVED** [review]
Nice! Thank you for your contribution 🙏

**Comment by zomars** [comment]
I don't have permissions to update this branch. Please rebase to main so we can marge. 🙏🏽

**Comment by miguelnietoa** [comment]
> I don't have permissions to update this branch. Please rebase to main so we can marge. 🙏🏽

Done!

**Pull request merged**